pym/portage/package/ebuild/config.py

```diff
@@ -1775,26 +1735,17 @@ def setcpv(self, mycpv, use_cache=None, mydb=None):
 			self, unfiltered_use, use, self.usemask,
 			portage_iuse, use_expand_split, self._use_expand_dict)
 
-		use_expand_iuses = {}
+		use_expand_iuses = dict((k, set()) for k in use_expand_split)
 		for x in portage_iuse:
 			x_split = x.split('_')
 			if len(x_split) == 1:
 				continue
 			for i in range(len(x_split) - 1):
 				k = '_'.join(x_split[:i+1])
 				if k in use_expand_split:
-					v = use_expand_iuses.get(k)
-					if v is None:
-						v = set()
-						use_expand_iuses[k] = v
-					v.add(x)
+					v = use_expand_iuses[k].add(x)
```

Comment:
Unnecessary assignment of `v` here.

Reply:
Yeah, looks like I forgot to remove it when I modified the `v=` line ;-).